Mathlib/Data/Int/Order/Basic.lean

```diff
@@ -121,6 +121,11 @@ lemma cast_mul_eq_zsmul_cast {α : Type*} [AddCommGroupWithOne α] :
     simp only [sub_mul, one_mul, cast_sub, ih, sub_zsmul, one_zsmul, ← sub_eq_add_neg, forall_const]
 #align int.cast_mul_eq_zsmul_cast Int.cast_mul_eq_zsmul_cast
 
+theorem natAbs_add_neg_pos_iff (i j : ℤ) : 0 < natAbs (i + -j) ↔ i ≠ j := by
+  rw [natAbs_pos, ne_eq, add_neg_eq_zero]
```

Comment:
Are there any theorem statements about `x + -y` in mathlib besides `sub_eq_add_neg`? Do you really need this variant?

Reply:
`sub_eq_add_neg` isn't a `@[simp]`, so my reasoning was that by doing so I'd enable some simps. To be fair I can do this transformation at call site and scrap this variant alltogether